assets/js/prosemirror/blocks.js

```diff
@@ -5,9 +5,29 @@ const prosemirrorBlocks = {
   text: nodes.text,
   heading: nodes.heading,
   doc: nodes.doc,
-  image: nodes.image
+  image: nodes.image,
+  html: {
+    inline: true,
+    attrs: {
+      html: {default: null}
+    },
+    group: 'inline',
```

Comment:
I'm not sure which of the configs impacts, but I trust that the html content should not be placed inline, instead it should be placed isolated from an existing paragraph and preferably not inside any.

That is because depending on the paragraph the user is, the content might be placed on a weird place if it gets inside the paragraph.

For me, the best would be the html content be placed on its own content isolated from any existing paragraph, so if the user paste the content it gets pasted after the paragraph.

Also I don't think that the html content should be impacted by any format someone do on the text, but maybe that is not viable to detect or prevent.

Reply:
I worry something like this might happen

```html
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam in lobortis libero. Mauris non lectus metus. Maecenas ut orci luctus, euismod justo at, tincidunt arcu. Phasellus volutpat iaculis magna, ut bibendum odio imperdiet sit amet. Proin malesuada egestas erat, ac dignissim quam posuere at. <embed>BLABLA</embed>Donec quis lectus sit amet neque consequat iaculis ut eget dui. Etiam ac enim lorem. Sed sodales nibh vitae convallis ultrices. Phasellus in justo consectetur, tincidunt dolor id, viverra lacus.</p>
```

When I think this should happen instead
```html
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam in lobortis libero. Mauris non lectus metus. Maecenas ut orci luctus, euismod justo at, tincidunt arcu. Phasellus volutpat iaculis magna, ut bibendum odio imperdiet sit amet. Proin malesuada egestas erat, ac dignissim quam posuere at.</p>

<embed>BLABLA</embed>

<p> Donec quis lectus sit amet neque consequat iaculis ut eget dui. Etiam ac enim lorem. Sed sodales nibh vitae convallis ultrices. Phasellus in justo consectetur, tincidunt dolor id, viverra lacus.
```

Reply:
it is only a div because my Sample app has it as a div, but if I change it to a span it will load as follows:

<img width="806" alt="Screen Shot 2021-06-08 at 11 20 25 AM" src="https://user-images.githubusercontent.com/20251602/121212623-88505300-c84b-11eb-9e3c-6135b2dce994.png">

Which makes sense if we wanted to put a span within a paragraph. It is up to the app to prove the data/html you want to load. as for the marks (bold, underling, etc...), It makes total sense to have it work, all it does is wrap your content with specific things.

Reply:
I do not think we can control where they want to place their content, if it is in a `<p>` then thats where it is. We cannot just close all tags thats your in and go up the root document and create an embed there. How would we know how far to go? That IMO would cause weird results, when you place some embedded html but it displays outside of where you inserted it

Reply:
yeah, it might have weird results depending how deep you are, indeed